hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenManagementSpecs.java

```diff
@@ -341,7 +510,7 @@ public HapiSpec wipeAccountFailureCasesWork() {
                             final var alias = key.hasECDSASecp256K1() ? key.getECDSASecp256K1() : key.getEd25519();
                             allRunFor(
                                     spec,
-                                    wipeTokenAccountWithAlias(unwipeableToken, alias, 1)
+                                    wipeTokenAccountWithAlias(unwipeableToken, "alias", 1)
```

Comment:
Is this change intentional?

Reply:
Yes, since I changed ` wipeTokenAccountWithAlias` to use String just like other `...WithAlias` methods 😃